arrays/arrays.c

```diff
@@ -23,9 +26,12 @@ Array *create_array (int capacity) {
   // Allocate memory for the Array struct
   Array *array = malloc(sizeof(Array));
   // Set initial values for capacity and count
-  array->capacity = capacity
-  array->count = -1
+  array->capacity = capacity;
+  array->count = 0;
   // Allocate memory for elements
+  array->elements = malloc(capacity * sizeof(char*));
+
```

Comment:
great work but make sure to eliminate this white space for good practice :), awesome job tho with the logic!